app/views/index.html

```diff
@@ -38,6 +38,30 @@ <h2 class="govuk-heading-m">Examples and documentation</h2>
           </a>
         </li>
       </ul>
+
+      {% from "hmcts-progress-bar/macro.njk" import hmctsProgressBar %}
```

Comment:
Note if HMCTS decided to change their implementation to follow the proposed nested folder, they would have to change their config here: https://github.com/hmcts/frontend/blob/824ccb0e3f6db3da91655e69df46a05acb2a4445/src/govuk-prototype-kit.config.json#L6

And then change this to `{% from "hmcts/components/progress-bar/macro.njk" import hmctsProgressBar %}`